src/index.ts

```diff
@@ -1322,7 +1396,11 @@ export abstract class PinejsClientCore<PinejsClient> {
 			method,
 		};
 
-		return this._request(opts);
+		const { retry = {} } = params;
```

Comment:
Can you merge this with the other destructuring above please, also I think the default should be `false` to play nicely:
```suggestion
		const { retry = false } = params;
```